app/assets/javascripts/admin_users.js

```diff
@@ -59,8 +59,14 @@
     }
   }
 
+  function on_password_change() {
+    let sendInformationField =  jQuery('.send-information');
```

Comment:
Use `var` here instead `let` since anything under `app/assets` is sadly not being transpiled to ES5 :(